src/um/objidl.rs

```diff
@@ -1,29 +1,113 @@
+// Copyright © 2016-2019 winapi-rs developers
```

Comment:
What I meant was that I intentionally removed copyright years because they cause a lot of churn and don't actually mean anything legally, so you should remove it too.

Reply:
Done. I'll keep that in mind for the future.